pkg/services/meta/notifications.go

```diff
@@ -626,3 +626,16 @@ func (m *Meta) handleEpochNotification(e uint64) error {
 
 	return nil
 }
+
+// NotifyObjectSuccess subscribes channel for object notification chain inclusion.
+// Channel must be read before subscription is made and writing to it must be
+// non-blocking.
+func (m *Meta) NotifyObjectSuccess(ch chan<- error, addr oid.Address) {
```

Comment:
error is never actually written to `ch`, i'd simplify to `chan<- struct{}`

Reply:
i do expect it to be more expressive in the future (like be possible to use VUB and to timeout inside meta service too, if no tx will be added to the chain) but not for now

Reply:
i'd extend when rly needed